src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -0,0 +1,467 @@
+
+
+
+  {% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/eSGbG9uGU0NXxXuHOtw7.png",
+    alt="Screenshot of the DevTools Network panel showing that the SXG 
+    and its certificate were loaded with no errors.",
+    width="696",
+    height="201"
+  %}
+
+
+
+## Serve signed exchanges using a `CanSignHttpExchanges` certificate
+
+  The instructions in this section explain how to serve SXGs using a
+  `CanSignHttpExchanges` certificate. Production use of SXGs requires a
+  `CanSignHttpExchanges` certificate.
+
+  These instructions are fairly similar to those for serving SXGs with a
+  self-signed certificate. For the sake of brevity, these instructions are
+  written with the assumption that you understand the concepts discussed in the
+  [Setup Signed Exchanges using a self-signed
+  certificate](/signed-exchanges-webpackager/#serve-sxgs-using-a-self-signed-certificate)
+  section.
+
+
+### Prerequisites
+
+  You have a `CanSignHttpExchanges` certificate. This
+  [page](https://github.com/google/webpackager/wiki/Certificate-Authorities) lists
+  the CAs that offer this type of certificate.
+
+
+### Generate a `CanSignHttpExchanges` certificate
+
+
+#### Prerequisites
+
+
+
+1. Install the
+   [`gen-certurl`](https://github.com/WICG/webpackage/blob/master/go/signedexchange/README.md)
+   tool.
+
+    ```shell
+    go get -v -u github.com/WICG/webpackage/go/signedexchange/cmd/gen-certurl
+    ```
+
+    `gen-certurl` is a tool that converts certificates to the certificate format
+    used by signed exchanges.
+
+
+
+2. Verify that `gen-certurl` has been installed correctly.
+
+    ```shell
+    gen-certurl --help
+    ```
+
+    This command should return information about its usage.
+
+
+#### Instructions
+
+
+
+1. Follow the steps 1 through 3 of [Generate a self-signed
+   certificate](/signed-exchanges-webpackager/#generate-a-self-signed-certificate).
+2. Convert `cert.pem` to the `application/cert-chain+cbor` format. 
+
+    ```shell
+    gen-certurl -pem cert.pem -ocsp &lt;(echo ocsp) > cert.cbor
+    ```
+
+    Certificates come in many formats. `cert.pem` is in the [PEM
+    format](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail). Certificates for
+    signed exchanges must be in the `application/cert-chain+cbor` format.
+
+    When using a self-signed certificate, `webpkgserver` automatically converts
+    the certificate indicated by the `PEMFile` option in `webpkgserver.toml` to
+    the `application/cert-chain+cbor` format. However, if you are using a
+    `CanSignHttpExchanges` certificate, you must generate the CBOR-encoded
+    certificate yourself.  
+
+
+### Setup up Web Packager for production use
+
+
+
+1. Follow steps 1 and 2 of [Setup the Web Packager Server for
+   testing](/signed-exchanges-webpackager/#setup-the-web-packager-server-for-testing).
+
+2. Open `webpkgserver.toml` with the editor of your choice and make the
+   following changes:
+    *   Change the line `PEMFile = cert.pem` to `PEMFile = cert.cbor`.
```

Comment:
This line should be changed. PEMFile should point to a PEM file that contains the concatenation of your site's SXG certificate and your CA's certificate.